sensor/tag_based_sfm_calibrator/README.md

```diff
@@ -0,0 +1,226 @@
+- The ground tags can be printed using normal paper and when attached via tape to the floor. The user needs to be careful not to move them during the experiment while talking.
+- The ground tags can be laminated for reuse, but the reflections caused by it can make it difficult or impossible to detect.
+  Auxiliary tag
+  Auxiliary tags are tags that do not have restrictions on their placement and use and were added to improve the observation distribution in the bundle adjustment graph. If it were not for the auxiliary tags, most of the tags would be on a single plane, save a few observations corresponding to the waypoints and the wheel tags.
+
+Considerations:
+
+- It is recommended to place them in positions and orientations without sufficient samples otherwise. A good examples are walls, for which an example is presented in Figure 5.
+
+### External camera
+
+The role of the external camera is to connect the sensors and their detections creating the graph required for bundle adjustment
+
+Considerations:
+
+- The camera must behave like a pinhole + distortion model. That is to say, the lenses must be "fixed" during the experiment.
+- The highest "real" resolution the better. So far we have tried with Nikon DSLRs (half frame) with moving and fixed lenses, and a simple point-and-shot camera.
+- In the pro-tips / recommendations section, more suggestions regarding the camera settings have been provided
+
+### Initial (external) camera intrinsics calibration board
+
+To use the external camera in bundle adjustment, it is necessary to know, to a certain degree, its intrinsics.
+Any board that can be used for this purpose is acceptable but circle-patterned chess boards and apriltags are supported directly by this tool.
+
+<table>
+  <tr>
+    <td><img src="../docs/images/tag_based_sfm_calibrator/waypoint_tag.jpg" alt="waypoint" height = 256px ></td>
+    <td><img src="../docs/images/tag_based_sfm_calibrator/wheel_tag.jpg" alt="wheel" width = 256px height = 256px></td>
+    <td><img src="../docs/images/tag_based_sfm_calibrator/ground_tag.jpg" alt="ground" width = 256px height = 256px ></td>
+    <td><img src="../docs/images/tag_based_sfm_calibrator/auxiliar_tag.jpg" alt="auxiliary"width = 256px height = 256px></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Figure 2. Waypoint tag</p></td>
+    <td><p style="text-align: center;">Figure 3. Wheel tag</p></td>
+    <td><p style="text-align: center;">Figure 4. Ground tag</p></td>
+    <td><p style="text-align: center;">Figure 5. Auxiliar tag</p></td>
+  </tr>
+</table>
+
+## References/External links
+
+[1] Jiunn-Kai (Bruce) Huang, Shoutian Wang, Maani Ghaffari, and Jessy W. Grizzle, "LiDARTag: A Real-Time Fiducial Tag System for Point Clouds," in IEEE Robotics and Automation Letters. Volume: 6, Issue: 3, July 2021.
+
+## Known issues/limitations
+
+- Our version of lidartag only supports the family `16h5`
```

Comment:
Please add , in the end of the sentences.

Reply:
Thanks. Addressed in [3797778](https://github.com/tier4/CalibrationTools/pull/164/commits/3797778c1b916f5f8400833bac4fb738330e4ca9)